be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp

```diff
@@ -12,8 +12,9 @@ Status AggregateBlockingSinkOperator::prepare(RuntimeState* state) {
     return _aggregator->open(state);
 }
 
-bool AggregateBlockingSinkOperator::is_finished() const {
-    return _is_finished;
+Status AggregateBlockingSinkOperator::close(RuntimeState* state) {
+    RETURN_IF_ERROR(_aggregator->close_one_operator(state));
```

Comment:
close/create_one_operator are confusing,  it's a refcount of close/create operations.  suggest to rename 
 them to:
1. ref()/unref(): referring to src/util/callback_closure.h, leveldb also use Ref/Unref.
2. AddRef()/Release(): referring to src/gutil/ref_counted.cc